fix: Add Ownership Rules 100K char limitation (#6013)
getsentry/sentry-docs: src/docs/product/issues/issue-owners/index.mdx

When you create a rule from the **Issue Details** page, you'll see some suggested paths and URLs based on the events in the issue. Note that [sentry.io](https://sentry.io) doesn't suggest tags.

#### Limitations

> We support Ownership Rules with a maximum of 100,000 characters. If your rule exceeds this size, we suggest using wildcard rules to consolidate multiple entries into a single one.
> 

Reviewer: ```suggestion
We support ownership rules with up to 100,000 characters. If your rule exceeds this size, we suggest using wildcard rules to consolidate multiple entries into one.

```
